sympy/physics/units/quantities.py

```diff
@@ -89,7 +89,7 @@ def _eval_is_constant(self):
         return self.scale_factor.is_constant()
 
     def _eval_Abs(self):
-        return self.__class__(self.name, self.dimension,
+        return Quantity(self.name, self.dimension,
```

Comment:
Well, I usually accept maintainers making small fixes (typos, docs, etc.) as long as it is not during discussion about design.